pom.xml

```diff
@@ -155,6 +155,35 @@
             <url>http://maven.dans.knaw.nl/releases</url>
         </pluginRepository>
     </pluginRepositories>
+    <build>
+        <plugins>
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-dependency-plugin</artifactId>
+                <version>2.8</version>
```

Comment:
"version" is redundant, as you are inheriting the parent's plugin declaration.

Reply:
done